tests/test_class_sh_void_ptr_capsule.cpp

```diff
@@ -0,0 +1,134 @@
+#include "pybind11_tests.h"
+
+#include <Python.h>
```

Comment:
We aren't suppose to include Python.h explicitly (it's included via PyBind11). Pybind is included via Pybind11_tests.h

Reply:
Done.